axiom/tests/test_txn.c

```diff
@@ -8537,6 +8537,33 @@ static void test_txn_log_configuration(void) {
   // clang-format on
 }
 
+static void test_nr_txn_add_php_package(void) {
+  char* json;
+  char* package_name1 = "Laravel";
+  char* package_version1 = "8.83.27";
+  char* package_name2 = "Slim";
+  char* package_version2 = "4.12.0";
+  nrtxn_t* txn = new_txn(0);
+
+  /*
+   * NULL parameters: ensure it does not crash
+   */
+  nr_txn_add_php_package(NULL, NULL, NULL);
```

Comment:
I would test for all combinations with a single `NULL` and otherwise valid parameters to make sure the first parameter being `NULL` doesn't shield other values being invalid having issues since the code probably checks for the first parameter being invalid and exits.

Reply:
good idea, added more tests: [b8d3820](https://github.com/newrelic/newrelic-php-agent/pull/754/commits/b8d3820c9de657c0f0be37ca566caf56db3bace6)